codec-mqtt/src/test/java/io/netty/handler/codec/mqtt/MqttReasonCodesTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2023 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.handler.codec.mqtt;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.Disconnect;
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.Auth;
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.PubAck;
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.PubRec;
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.PubRel;
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.PubComp;
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.SubAck;
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.UnsubAck;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+// ./mvnw -B -ntp --file pom.xml verify -pl codec-mqtt
+
```

Comment:
This comment is unnecessary.

Reply:
Fixed by 3a9ff0ddc2fdf77a688e231eef9f5f1c8210e65e